tracing/src/test/java/org/axonframework/extensions/tracing/TracingCommandGatewayTest.java

```diff
@@ -72,19 +72,22 @@ public void testSendWithCallback() {
 
         List<MockSpan> mockSpans = mockTracer.finishedSpans();
         assertThat(mockSpans.size(), is(1));
-        assertThat(mockSpans.get(0).operationName(), is("sendCommandMessage"));
+        assertThat(mockSpans.get(0).operationName(), is("sendMyCommand"));
 
         mockTracer.scopeManager().active().close();
         assertThat(mockTracer.scopeManager().active(), nullValue());
     }
 
+    private static class MyCommand {
```

Comment:
Super-nit: would you mind moving this class to the bottom of the file?

Reply:
check